### PR TITLE
Blog post about Strata+Hadoop World Tutorial

### PR DESCRIPTION
**Pull request by eljefe6a**
Added blog post about Strata+Hadoop World tutorial. Added Jesse Anderson as an author. Added the images for the blog post.

### CODE REVIEW

**Comment by nssalian** [comment]
Looks good.  Will wait for another set of eyes to help review. Thanks @eljefe6a

File: content/beam/update/2016/10/11/strata-hadoop-world-and-beam.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
```
Comment (francesperry) [inline content/beam/update/2016/10/11/strata-hadoop-world-and-beam.html]: FYI -- in the future, please drop the content directory from the PR. (Merging autogenerated content gets gross, so the person who does the merge will do the final generate. Instructions in contribution guide are getting updated in pull/42.)

File: _posts/2016-10-12-strata-hadoop-world-and-beam.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<img src="{{ "/images/blog/IMG_20160927_170956.jpg" | prepend: site.baseurl }}" alt="Exercise time">
+
+If you want to take a look at the tutorial materials, we’ve put them up [on GitHub](https://github.com/eljefe6a/beamexample). This includes the [actual slides](https://github.com/eljefe6a/beamexample/blob/master/BeamTutorial/slides.pdf) as well as the [exercises](https://github.com/eljefe6a/beamexample/tree/master/BeamTutorial/src/main/java/org/apache/beam/examples/tutorial/game) that we covered. If you’re looking to learn a little about Beam, this is a good way to start. The exercises are based on an imaginary mobile game and the data needs processing and are based on code in the [Beam examples directory](https://github.com/apache/incubator-beam/tree/master/examples/java8/src/main/java/org/apache/beam/examples/complete/game). The code has TODOs for where you need to fill in code or there are full sample solutions to look over our code. You can run these examples on your own machine.
```
Comment (francesperry) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: Run on sentence of some sort? "...exercises are based on an imaginary mobile game and the data needs processing and are based on..."

Comment (francesperry) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: Maybe add something about multiple runners being supported here (as well as at the end where you talk about the demo).

File: _posts/2016-10-12-strata-hadoop-world-and-beam.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+We heard [loud and clear](https://twitter.com/jessetanderson/status/781124173108305920) from Beam users that great runner support is crucial to adoption. We have great Apache Flink support. During the conference we had some more volunteers offer their help on the Spark runner.
+
+On management and thought leader side, Beam went from “what’s Beam?” at previous conferences to “I’m interested in Beam.” or “I’ve formed an informed opinion on Beam.” at this conference. This is one of the metrics I look for in early technology adoption.
+
+<img src="{{ "/images/blog/IMG_20160927_170455.jpg" | prepend: site.baseurl }}" alt="So much brainpower answering questions">
+
+We rounded out the tutorial with live demonstrations of Beam running on Apache Spark, Apache Flink, the local runner, and DataFlow runner. Then, we brought in the big brainpower and had a Q and A session.
+
+If you’re attending a conference, we encourage you to look for a Beam session. If you want to use these materials to give your own Beam talk, we’re happy to help you. You can reach out to us on the [user mailing list](http://beam.incubator.apache.org/use/mailing-lists/).
```
Comment (francesperry) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: talk -> tutorial?

(Maybe also mention the separate talk resources? beam.incubator.apache.org/learn/presentation-materials/ )

**Comment by francesperry** [comment]
Reviewer: @francesperry

Nice, Jesse! Left a couple of suggestions. Will plan to merge tomorrow since that's the date you put on it. (Though I guess it's already tomorrow in some places... ;-)

File: _posts/2016-10-12-strata-hadoop-world-and-beam.md (hunk 3 above)
Comment (eljefe6a) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: Done

File: _posts/2016-10-12-strata-hadoop-world-and-beam.md (hunk 2 above)
Comment (eljefe6a) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: Done

Comment (eljefe6a) [inline _posts/2016-10-12-strata-hadoop-world-and-beam.md]: Fixed

**Pull request closed without merging**